### DISCUSSION TITLE
What's the benefit of using this over GA, vue-gtag etc?

### DISCUSSION THREAD

**Discussion by notflip** [gtm-support/vue-gtm#352] [Q&A]
Is there a difference, It seems like a lot of features are similar.

**Comment by Shinigami92** [comment]
I'm highly sure the is no big difference except of how it works, and strategies of usage.

It is more the history:

https://github.com/mib200/vue-gtm (the original before I took over the maintaining) is around 6 years old, were as https://github.com/MatteoGabriele/vue-gtag is just around 5 years old

So it might be more the question, why was there ever a vue-gtag developed beside vue-gtm 🤷
I don't know and I'm even not that long in the Vue and Open Source Ecosystem 😅

That said: if vue-gtag fits your need, PLEASE use vue-gtag!!! because I have far more other priorities as to maintain this repo here. I don't know the contributor(s) of vue-gtag, but it might be that they are far more active then I am (at least for this repo 😉)
I used vue-gtm myself in my previous company and so I needed the maintenance so I took it over, but nowadays I don't need it myself and I have nearly zero knowledge of Google Tag Manager or Google Analytics myself 😕

I just fix bugs and security issues if there are no workarounds, and also I can try to help guiding to implement meaningful features. So I currently don't just drop the repo, I'm still alive 🙂

**Reply by notflip** [reply]
Thanks for getting back, vue-gtag works (sortoff), for example when I add a querystring to a page `?ref=mail` it doesn't get picked up by vue-gtag, it's only sending the name of the route. Is this something, that would work with this repo?

**Reply by Shinigami92** [reply]
This repo only works with IDs following the schema `GTM-xxxxxx`
If you have GA or GU or whatever there is, you will be out of luck by design

---

Apart from this, with this repo I assume you would need to call trackEvent or something like this on your own, but then can do whatever you want to send
